services/ui/ws/display.cc

```diff
@@ -77,14 +77,25 @@ Display::~Display() {
 }
 
 void Display::Init(const display::ViewportMetrics& metrics,
-                   std::unique_ptr<DisplayBinding> binding) {
+                   std::unique_ptr<DisplayBinding> binding,
+                   const ServerWindow* parent_window) {
```

Comment:
another suggestion:

we have this display::ViewportMetrics parameter, that we have already modified for our needs, in order to pass information down to ozone layers. It also contains a comment saying: "this struct should be renamed to XXX" or so.

can we put this parent display in the structure somehow to pass it to ozone?